### PR TITLE
More generous timeout in file upload sample spec

### LINKED ISSUE #20727: FAILED: FileUploadDirectivesExamplesSpec
https://jenkins.akka.io:8498/job/akka-nightly/3387/consoleFull

```
[info] - uploadedFile *** FAILED *** (1 second, 69 milliseconds)
[info]   Request was neither completed nor rejected within 1 second (FileUploadDirectivesExamplesSpec.scala:35)
[info]   org.scalatest.exceptions.TestFailedException:
[info]   at akka.http.scaladsl.testkit.TestFrameworkInterface$Scalatest$class.failTest(TestFrameworkInterface.scala:24)
[info]   at docs.http.scaladsl.server.RoutingSpec.failTest(RoutingSpec.scala:12)
[info]   at akka.http.scaladsl.testkit.RouteTestResultComponent$RouteTestResult.failNeitherCompletedNorRejected(RouteTestResultComponent.scala:94)
[info]   at akka.http.scaladsl.testkit.RouteTestResultComponent$RouteTestResult.rawResponse(RouteTestResultComponent.scala:59)
[info]   at akka.http.scaladsl.testkit.RouteTestResultComponent$RouteTestResult.response(RouteTestResultComponent.scala:40)
[info]   at akka.http.scaladsl.testkit.RouteTest$class.response(RouteTest.scala:62)
[info]   at docs.http.scaladsl.server.RoutingSpec.response(RoutingSpec.scala:12)
[info]   at akka.http.scaladsl.testkit.RouteTest$class.status(RouteTest.scala:80)
[info]   at docs.http.scaladsl.server.RoutingSpec.status(RoutingSpec.scala:12)
[info]   at docs.http.scaladsl.server.directives.FileUploadDirectivesExamplesSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(FileUploadDirectivesExamplesSpec.scala:35)
[info]   at docs.http.scaladsl.server.directives.FileUploadDirectivesExamplesSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply(FileUploadDirectivesExamplesSpec.scala:35)
[info]   at docs.http.scaladsl.server.directives.FileUploadDirectivesExamplesSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply(FileUploadDirectivesExamplesSpec.scala:35)
[info]   at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
[info]   at akka.http.scaladsl.testkit.RouteTest$$anonfun$check$1.apply(RouteTest.scala:57)
[info]   at akka.http.scaladsl.testkit.RouteTest$$anonfun$check$1.apply(RouteTest.scala:57)
[info]   at akka.http.scaladsl.testkit.RouteTestResultComponent$RouteTestResult.$tilde$greater(RouteTestResultComponent.scala:51)
[info]   at 
```

### PR DESCRIPTION
**Pull request by johanandren**
(Which touches filesystem)
Fixes #20727

### CODE REVIEW

File: akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala [hunk 1]
```diff
@@ -99,9 +99,10 @@ class DebuggingDirectivesExamplesSpec extends RoutingSpec {
   }
   "logRequestResultWithResponseTime" in {
 
-    def akkaResponseTimeLoggingFunction(loggingAdapter: LoggingAdapter,
-                                        requestTimestamp: Long,
-                                        level: LogLevel = Logging.InfoLevel)(req: HttpRequest)(res: Any): Unit = {
+    def akkaResponseTimeLoggingFunction(
+      loggingAdapter:   LoggingAdapter,
+      requestTimestamp: Long,
+      level:            LogLevel       = Logging.InfoLevel)(req: HttpRequest)(res: Any): Unit = {
```
Comment (patriknw) [inline akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala]: 👍

**Comment by patriknw** [comment]
LGTM

File: akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala (hunk 1 above)
Comment (johanandren) [inline akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala]: The autoindent-sbt-plugin deserves all the praise ;)

**Comment by drewhk** [comment]
LGTM

**Pull request merged**